tsconfig.spec.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "./tsconfig.json",
```

Comment:
this makes no sense

Reply:
true, let me check

Reply:
actually, this is correct, why does it make no sense?

Reply:
it's extending itself

Reply:
ah sorry, no it's not